### ISSUE TITLE
Incompatible with Time and Wind

### ISSUE THREAD

**Issue by plantiac** [Steveplays28/realisticsleep#49]
### Versions
Minecraft 1.19.2
Fabric 0.14.22
Time and Wind 1.4.3
Realistic Sleep 1.8.3

### Details
Realistic Sleep 'snaps' adjusted tick time back to vanilla tick time (if that makes sense).

**Comment by Steveplays28** [comment]
Hi, the most important features of Time and Wind, such as slowing time down are already implemented in Realistic Sleep. Use the tick delay option in the config.
Time and Wind is marked as `breaks` in the [`fabric.mod.json`,](https://github.com/Steveplays28/realisticsleep/blob/main/src/main/resources/fabric.mod.json), I'll add compatibility info to the readme.

**Comment by plantiac** [comment]
Hey, I haven't got to configure realistic sleep yet and assumed the time manipulation only kicks in while sleeping. Thanks for the info

**Comment by plantiac** [comment]
I know this doesn't fit in this issue but could you provide more information (in the `README.md` for other players as well) on what exactly gets sped up during sleep? Do crops also get sped up or only when the chunk tick speed multiplier is turned on?

**Comment by Steveplays28** [comment]
I can add some more documentation. As for your question, crops only get sped up when the chunk tick speed multiplier is turned on. I might look into a better way to handle that for crops specifically.

**Comment by Steveplays28** [comment]
Hi again plantiac, I'm curious as to how you got this to run at all, did you use a Fabric/Quilt loader override?
Time and Wind is marked as a conflict and shouldn't allow the game to load when ran together with Realistic Sleep.

**Comment by plantiac** [comment]
Hey, I had both mods normally installed and there was no conflict pop-up hindering me from launching the game. I'm using your mod solely now, since includes taw funcionality and some other features as well.

**Comment by Steveplays28** [comment]
Thanks for the quick response. I found out why: I added Time and Wind as a soft conflict instead of a hard breakage, I thought I added it as a hard breakage, my bad.

**Comment by MrGreenTea** [comment]
> Hi, the most important features of Time and Wind, such as slowing time down are already implemented in Realistic Sleep. Use the tick delay option in the config. Time and Wind is marked as `breaks` in the [`fabric.mod.json`,](https://github.com/Steveplays28/realisticsleep/blob/main/src/main/resources/fabric.mod.json), I'll add compatibility info to the readme.

Is there some documentation on how to use `tickDelay` / what the value exactly means and does?

**Comment by Steveplays28** [comment]
`tickDelay` allows you to delay the day/night cycle by a certain amount of ticks, every tick.
I might rework the settings in the future to be more clear.

**Comment by MrGreenTea** [comment]
Does this mean that a tickdelay of 1 means that the day and night are twice as long because every tick gets one extra tick? Does this allow fractional scaling of the day length? And does this mean day and night will both be longer?

**Comment by Steveplays28** [comment]
Yes, with a `tickDelay` of `1` it'll mean the cycle is twice as long. Fractional scaling isn't implemented yet. Might rework all settings to allow fractions and options less than 1.
There's no individual settings for day and night yet, both are adjusted at the same time.
